src/types.js

```diff
@@ -160,8 +160,8 @@ export type EventData = {
   customerId?: string,
   externalId?: string,
   sessionId?: string,
-  type: string,
-  context: string,
+  type: string, // TODO: shouldn't be an enum?
```

Comment:
Yep, I think I was just too lazy to write it. The full list of valid event types is:
```json
[ "abandonedCart", "addedCompare", "addedProduct", "addedWishlist", "campaignBlacklisted", "campaignBounced", "campaignLinkClicked", "campaignMarkedSpam", "campaignOpened", "campaignSent", "campaignSubscribed", "campaignUnsubscribed", "changedSetting", "clickedLink", "closedTicket", "completedOrder", "eventInvited", "eventParticipated", "formCompiled", "genericActiveEvent", "genericPassiveEvent", "loggedIn", "loggedOut", "openedTicket", "orderShipped", "removedCompare", "removedProduct", "removedWishlist", "repliedTicket", "reviewedProduct", "searched", "serviceSubscribed", "serviceUnsubscribed", "viewedPage", "viewedProduct", "viewedProductCategory" ]
```

(copied from [here](https://github.com/contactlab/contacthub-sdk-dotnet/blob/d816aed02244099a070d4c7422c4eb8bb1798c89/ClassBuilder/ClassBuilder/bin/Debug/JsonSchema/docs.schema.enums.json), should be up to date)